teams/models.py

```diff
@@ -17,11 +17,18 @@ def generate_teams(event_id):
         from event.models import UserEvent
         users_events = UserEvent.objects.filter(eventID=event_id)
         team_size = len(users_events) // 2
-        team1 = Teams(name="Team1")
+        team1 = Teams(name=f"{event_id}Team1")
```

Comment:
Consider a seperator between the event_id and Team. 
```suggestion
        team1 = Teams(name=f"{event_id}-Team1")
```